bot/models/user.py

```diff
@@ -0,0 +1,41 @@
+from aiogram import types
+from peewee import *
+
+from bot.models.core import BaseModel
+from bot.models.fraction import Fraction
+from bot.utils import aiowrap
+
+
+class User(BaseModel):
+    user_id = IntegerField(unique=True)
+    username = CharField(null=True)
+    first_name = CharField(null=True)
+    last_name = CharField(null=True)
+    fraction = ForeignKeyField(Fraction, backref="member", null=True)
+    locale = CharField(max_length=3, null=True)
+    start_conversation = TimestampField()
+
+    @classmethod
+    def create_user(cls, tg_user: types.User):
+        cls.create(
+            user_id=tg_user.id,
```

Comment:
if we left these methods like create_user and get_user we need to create method like params because DRY
something like that
```
def params(tg_user):
  {
    user_id=tg_user.id,
    username=tg_user.username,
    first_name=tg_user.first_name,
    flast_name=tg_user.last_name,
    locale=tg_user.locale,
  }
```